commands/deploy.js

```diff
@@ -42,6 +42,7 @@ module.exports = new Command("deploy")
   .description("deploy code and assets to your Firebase project")
   .option("-p, --public <path>", "override the Hosting public directory specified in firebase.json")
   .option("-m, --message <message>", "an optional message describing this deploy")
+  .option("-f, --force", "delete Cloud Functions missing from the current working directory")
```

Comment:
Maybe mention something about "bypassing prompt" or "without confirmation" in the description? Since the regular deploy command without the flag will also delete functions, the difference is that it will prompt the user for confirmation.

Reply:
Done.